Lib/test/test_argparse.py

```diff
@@ -1934,6 +1934,30 @@ def test_required_subparsers_default(self):
         subparsers.add_parser('run')
         self._test_required_subparsers(parser)
 
+    def test_required_subparsers_no_destination_error(self):
+        parser = ErrorRaisingArgumentParser()
+        subparsers = parser.add_subparsers()
+        subparsers.add_parser('foo')
+        subparsers.add_parser('bar')
+        with self.assertRaises(ArgumentParserError) as excinfo:
+            parser.parse_args(())
+        self.assertRegex(
+            excinfo.exception.stderr,
+            'error: the following arguments are required: {foo,bar}\n$'
+        )
+
+    def test_wrong_argument_subparsers_no_destination_error(self):
+        parser = ErrorRaisingArgumentParser()
+        subparsers = parser.add_subparsers()
+        subparsers.add_parser('foo')
+        subparsers.add_parser('bar')
+        with self.assertRaises(ArgumentParserError) as excinfo:
+            parser.parse_args(('baz',))
+        self.assertRegex(
+            excinfo.exception.stderr,
+            r"error: argument {foo,bar}: invalid choice: 'baz' \(choose from 'foo', 'bar'\)\n$"
```

Comment:
"foo, bar" is duplicated. And this error message looks confusing to me. What it means?

Reply:
yeah I didn't really know what to do for this, it's essentially an anonymous positional choices argument.  Personally I'd love a default `dest='command'` but I imagine that could cause some incompatibility?

Then this would read something like `error: argument command: invalid choice ...`